pkg/service/publish_results.go

```diff
@@ -223,12 +223,8 @@ func (mw logmw) PublishResults(ctx context.Context, nodeKey string, results []di
 				continue
 			}
 
-			// Log queries that took more than 5 seconds
-			if r.QueryStats.WallTimeMs < 5000 {
-				continue
-			}
-			mw.knapsack.Slogger().Log(ctx, slog.LevelWarn,
-				"noticed long-running query",
```

Comment:
do we still find the long running piece to be useful? just wondering if adding a `"long_running": r.QueryStats.WallTimeMs > 5000` would make spotting these a little easier for debugging

Reply:
Oh yeah, good point, I was thinking about cloud log and not scanning a flare. I'll update!

Reply:
Done! Thank you